### ISSUE TITLE
helm-chart

### ISSUE THREAD

**Issue by davidkarlsen** [yahoo/k8s-namespace-guard#1]
Do you plan to provide a helm-chart for installation?

**Comment by mdhume** [comment]
@davidkarlsen Yes, we can provide a helm-chart for this webhook. Thanks for opening an issue we will provide one shortly.
